src/components/molecules/OcTable/OcTable.spec.js

```diff
@@ -195,15 +195,30 @@ describe("OcTable", () => {
     expect(wrapper.findAll(".oc-table-highlighted").length).toEqual(2)
   })
 
-  it("adds data-file-id for rows", () => {
+  it("adds data-item-id for rows", () => {
     const wrapper = shallowMount(Table, {
       propsData: {
         fields,
         data,
         highlighted: [],
       },
     })
-    expect(wrapper.html().indexOf("data-file-id")).toBeGreaterThan(-1)
+    console.log(wrapper.html())
```

Comment:
Dev leftover? 🐶

Reply:
Same question from me 😁 Otherwise looks good..

Reply:
as usual, get things out fast to test ci and forget to remove those leftovers :( sorry guys, cleaned